R/utilities-molecule-plots.R

```diff
@@ -34,7 +38,7 @@
     scale_alpha_manual(
       values = if (mapLabels$lloq != "legendLabels") {
         # ensure that no matter the alpha setting is, the computed level is between 0 and
-        c(0.618*aestheticValues$alpha, aestheticValues$alpha)
```

Comment:
Could you define a constant for this magic number? And also add a short note why `0.618`.

Reply:
This is because according to my testing alpha and ~`0.6*alpha` can be differentiated by the eye, not matter the value of alpha between .2 and 1. It also ensure that the most transparent point stays visible event with low alpha value.

To make it less "arbitrary" I chose `1/golden ratio` = 0.618 as magic number.
Here is a preview of alpha and `0.618*alpha` levels:

![image](https://user-images.githubusercontent.com/34234913/229738350-7ec61fa0-c33d-4edd-b6a8-fff5c8c71a03.png)

```r
tibble(x = c(rep("y", 5), rep(".618*y", 5)), 
        y = c(seq(0.2, 1,by=0.2), seq(0.2, 1,by=0.2)), 
        alpha = c(seq(0.2, 1,by=0.2), 0.618* seq(0.2, 1,by=0.2))) %>% 
    ggplot(aes(fct_rev(as.factor(x)), y, alpha = alpha)) + 
    geom_point(size = 10) + 
    scale_alpha_identity() + 
    labs(x = "", y = "alpha") +
    theme_minimal()
```